scripts/dts/edtlib.py

```diff
@@ -1806,6 +1826,19 @@ def enum(self):
         "See the class docstring"
         return self._raw.get("enum")
 
+    @property
+    def enum_tokenizable(self):
+        "See the class docstring"
+        if not hasattr(self, '_enum_tokenizable'):
+            if self.type != 'string' or self.enum is None:
+                self._enum_tokenizable = False
+            else:
+                enum = self.enum
+                self._enum_tokenizable = all(
```

Comment:
Does something validate that the enum values are unique?

If we have a "token" variant that includes transformations, that would also be a condition for tokenizability so "a b" and "a-b" don't both map to `a_b`.